spec/features/put_routes_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 describe "put routes", type: :feature do
   let(:id) { Task.last.task_id }
-  scenario "update a task" do
-
-    visit "/todolist?id=#{id}"
 
-    fill_in "title", with: "App in test mode"
-    fill_in "start", with: "23/12/2015 12:00:00"
+  scenario "update a task" do
+    page.driver.put "/todolist",  { id: id, title: "App in test mode", start: "23/12/2015 12:00:00" }
```

Comment:
Line is too long. [101/80]<br>Redundant curly braces around a hash parameter.<br>Unnecessary spacing detected.